mindsdb/integrations/handlers/openai_handler/constants.py

```diff
@@ -17,4 +17,6 @@
                 tuple(f'code-search-{model}-text-001' for model in COMPLETION_LEGACY_BASE_MODELS) + \
                 tuple(f'code-search-{model}-code-001' for model in COMPLETION_LEGACY_BASE_MODELS)
 
-ALL_MODELS = list(set(CHAT_MODELS + COMPLETION_MODELS + COMPLETION_LEGACY_MODELS + EMBEDDING_MODELS))  # noqa
\ No newline at end of file
+IMAGE_MODELS = ('dall-e-3')
```

Comment:
This should be a tuple:

```
IMAGE_MODELS = ('dall-e-3',)
```

Notice that pesky little comma at the end which implies the type is `tuple` 👀

Reply:
Fixed. Tried it out and everything seems to be working.